_examples/document/conformance_attribute/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+
+package main
+
+import (
+	"github.com/unidoc/unioffice/document"
+	st "github.com/unidoc/unioffice/schema/soo/ofc/sharedTypes"
+)
+
+func main() {
+	doc, err := document.Open("document.docx")
+	if err != nil {
+		panic(err)
+	}
+	doc.SetConformance(st.ST_ConformanceClassTransitional)
```

Comment:
Would it be nicer to just set doc.SetStrict(true)  default is transitional I think always for word files.
I am not sure what the unset option means.